src/components/Footer/Footer.js

```diff
@@ -198,9 +317,19 @@ const styles = StyleSheet.create({
     borderBottomWidth: 1,
     borderBottomColor: COLORS.gray
   },
+  buttonShareContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingLeft: 0,
+    paddingRight: 0
+  },
+  textDM: {
+    marginLeft: 4,
+    fontWeight: '500'
+  },
   leftGroupContainer: {
     flexDirection: 'row',
-    alignItems: 'center'
+    alignItems: 'flex-start'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added for the DM button and the left group container. These styles seem appropriate for the new elements added to the component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->